packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -67,6 +71,7 @@ export class AtomicSearchBox implements AtomicSearchBoxOptions {
   constructor() {
     this.combobox = new Combobox({
       id: this._id,
+      strings: this.strings,
```

Comment:
I wasn't sure what the best way to access i18n inside a non-component class was. I'm not sure if this is the best practice @ThibodeauJF?

Reply:
Not sure, but if it works correctly it should be a good way to do it.

in the index.html script try this:
```
setTimeout(() => (searchInterface.language = 'fr'), 1000);
```

if the language change re-renders the strings correctly, it's good, if it doesn't, we'll have to find another way